rocky/reports/views.py

```diff
@@ -68,6 +69,10 @@ def setup(self, request, *args, **kwargs):
         self.yielded_report_types = get_report_types_for_oois(self.selected_oois)
         self.selected_report_types = request.GET.getlist("report_type", [])
 
+    def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        is_future_date(request, self.valid_time)
```

Comment:
This call is odd. The function name implies it's a getter, but you're not doing anything with the returned value

Reply:
agreed, is_future_date should return, and hijack the request not redirect by itself.